test/module.test.js

```diff
@@ -1762,3 +1762,66 @@ describe('generate with prefix strategy', () => {
     expect(canonicalLink?.getAttribute('href')).toBe('nuxt-app.localhost/en')
   })
 })
+
+describe('Configuration with locale fallback decision map', () => {
+  let nuxt
```

Comment:
Please annotate the type with:

```
  /** @type {Nuxt} */
```

like in other cases.

Reply:
Job's done